pkg/machine/e2e/set_test.go

```diff
@@ -139,8 +141,9 @@ var _ = Describe("podman machine set", func() {
 	})
 
 	It("set user mode networking", func() {
-		SkipIfNotWindows("Setting user mode networking is only honored on Windows")
-
+		if testProvider.VMType() != machine.WSLVirt {
```

Comment:
Does this test now work on Linux or other non windows environments?

Reply:
No, it is only for WSL